stl/inc/span

```diff
@@ -356,6 +349,10 @@ inline constexpr bool _Is_span_compatible_range = conjunction_v<
 // CLASS TEMPLATE span
 template <class _Ty, size_t _Extent = dynamic_extent>
 class span : public _Span_extent_type<_Ty, _Extent> {

```

Comment:
Since we no longer re-export any of `_Span_extent_type` in the interface of `span` this can now be
```suggestion
class span : private _Span_extent_type<_Ty, _Extent> {
```

Reply:
Done